src/SourceBuild/patches/source-build-reference-packages/0001-Enable-use-of-RestoreConfigFile-property-in-source-b.patch

```diff
@@ -0,0 +1,32 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nikola Milosavljevic <nikolam@microsoft.com>
+Date: Wed, 31 Jan 2024 16:53:44 +0000
+Subject: [PATCH] Enable use of RestoreConfigFile property in source-build
+
+Backport: https://github.com/dotnet/source-build-reference-packages/pull/880
+---
+ eng/SourceBuild.props | 4 +++-
+ 1 file changed, 3 insertions(+), 1 deletion(-)
+
+diff --git a/eng/SourceBuild.props b/eng/SourceBuild.props
+index 03a4893c..190c6ba7 100644
+--- a/eng/SourceBuild.props
++++ b/eng/SourceBuild.props
+@@ -12,6 +12,8 @@
+           BeforeTargets="Execute">
+     <PropertyGroup>
+       <LocalNuGetPackageCacheDirectory Condition="'$(LocalNuGetPackageCacheDirectory)' == ''">$(CurrentRepoSourceBuiltNupkgCacheDir)</LocalNuGetPackageCacheDirectory>
++      <_AdditionalDependencyProjectsBuildArgs />
```

Comment:
nit: you don't need this in msbuild. You can refer to an undefined property.

Reply:
Thanks - backport PR was already merged.